pcsx2/gui/GlobalCommands.cpp

```diff
@@ -70,10 +70,13 @@ namespace Implementations
 		g_Conf->EmuOptions.GS.FrameSkipEnable = !g_Conf->EmuOptions.GS.FrameSkipEnable;
 		SetGSConfig().FrameSkipEnable = g_Conf->EmuOptions.GS.FrameSkipEnable;
 
-		if( EmuConfig.GS.FrameSkipEnable )
+		if( EmuConfig.GS.FrameSkipEnable ) {
+			OSDlog( wxColor(L"RED"), "(FrameSkipping) Enabled : FrameDraws=%d, FrameSkips=%d", g_Conf->EmuOptions.GS.FramesToDraw, g_Conf->EmuOptions.GS.FramesToSkip );
```

Comment:
Yes but I think you can take the hypothesis that screen info must be printed on the console too. So you need
* a function that print in the console
* a function that print both in the screen and the console